pkg/controllers/managementuser/pspdelete/pspdelete.go

```diff
@@ -25,6 +29,18 @@ type handler struct {
 
 func Register(ctx context.Context, userContext *config.UserContext) {
 	starter := userContext.DeferredStart(ctx, func(ctx context.Context) error {
+		clusterName := userContext.ClusterName
+		logrus.Infof("Checking cluster [%s] compatibility before registering pspdelete controller.", clusterName)
+		clusterLister := userContext.Management.Management.Clusters("").Controller().Lister()
+		err := podsecuritypolicy.CheckClusterVersion(clusterName, clusterLister)
+		if err != nil {
+			if errors.Is(err, podsecuritypolicy.ErrClusterVersionIncompatible) {
+				logrus.Infof("%v - will not register pspdelete controller for cluster [%s].", err, clusterName)
+				return nil
+			}
+			return fmt.Errorf("unable to parse version of cluster %s: %w", clusterName, err)
+		}
+		logrus.Infof("Cluster [%s] is compatible with PSPs, will run pspdelete controller.", clusterName)
```

Comment:
We do not usually log when controllers start, so this can be debug or even trace.
```suggestion
		logrus.Debugf("Cluster [%s] is compatible with PSPs, will run pspdelete controller.", clusterName)
```